hovercontrol/hovercontrol.example.toml

```diff
@@ -0,0 +1,20 @@
+# The serial port connected to the right side of the hoverboard.
```

Comment:
hovercontrol/hovercontrol.toml.example seems more traditional (not hugely fussed though)

Reply:
`hovercontrol.example.toml` means that it gets recognised at TOML for syntax highlighting, which is useful.